windows/file_based_service_bindings_win.go

```diff
@@ -0,0 +1,89 @@
+package windows
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	. "github.com/cloudfoundry/cf-acceptance-tests/cats_suite_helpers"
+	"github.com/cloudfoundry/cf-acceptance-tests/helpers/app_helpers"
+	"github.com/cloudfoundry/cf-acceptance-tests/helpers/random_name"
+	services_test "github.com/cloudfoundry/cf-acceptance-tests/services"
+	"github.com/cloudfoundry/cf-test-helpers/v2/cf"
+	"github.com/cloudfoundry/cf-test-helpers/v2/generator"
+	"github.com/cloudfoundry/cf-test-helpers/v2/helpers"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = FileBasedServiceBindingsDescribe("File Based Service Bindings", WindowsLifecycle, func() {
+	var appName, serviceName string
+
+	getEncodedFilepath := func(serviceName string, fileName string) string {
+		path := fmt.Sprintf("/etc/cf-service-bindings/%s/%s", serviceName, fileName)
+		return strings.Replace(path, "/", "%2F", -1)
+	}
+
+	checkFileContent := func(fileName string, content string) {
+		curlResponse := helpers.CurlApp(Config, appName, "/file/"+getEncodedFilepath(serviceName, fileName), "-L")
+		Expect(curlResponse).Should(ContainSubstring(content))
+	}
+
+	getServiceInstanceGuid := func(serviceName string) string {
```

Comment:
This is quite some coding copied from `file_based_service_bindings.go`. Could you extract at least these two function to a common helper file?
`getServiceInstanceGuid`
`getServiceBindingGuid`
E.g. to `helpers/services/service_helpers.go`